main.cpp

```diff
@@ -50,108 +50,136 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
-
 //  1)
-//  Noun:
-//  action 1:
+//  Noun: shape
+//  action 1: the shape rotates
+shape.rotate();
 
-//  action 2:
+//  action 2: the shape calculates its area
+shape.calculateArea();
 
-//  action 3:
+//  action 3: the shape resizes
+shape.resize();
 
 //  2)
-//  Noun:
-//  action 1:
+//  Noun: dog
+//  action 1: the dog barks
+dog.bark();
 
-//  action 2:
+//  action 2: the dog bites
+dog.bite();
 
-//  action 3:
+//  action 3: the dog sleeps
+dog.sleep();
 
 //  3)
-//  Noun:
-//  action 1:
+//  Noun: audioPlayer
+//  action 1: the audio player plays
+audioPlayer.play()
 
-//  action 2:
+//  action 2: the audio player pauses
+audioPlayer.pause();
 
-//  action 3:
+//  action 3: the audio player stops
+audioPlayer.stop();
 
 //  4)
-//  Noun:
-//  action 1:
+//  Noun: calendar
+//  action 1: the calendar gets next meeting
+calendar.getNextMeeting();
 
-//  action 2:
+//  action 2: the calendar schedules a meeting
+calendar.scheduleMeeting();
 
-//  action 3:
+//  action 3: the calendar alerts
+calendar.alert();
 
 //  5)
-//  Noun:
-//  action 1:
+//  Noun: warrior
+//  action 1: the warrior choses a weapon
+warrior.chooseWeapon();
 
-//  action 2:
+//  action 2: the warrior aims
+warrior.aim();
 
-//  action 3:
+//  action 3: the warrior fights
+warrior.fight();
 
 //  6)
-//  Noun:
-//  action 1:
+//  Noun: game
+//  action 1: the game starts
+game.start();
 
-//  action 2:
+//  action 2: the game moves to the next turn
+game.nextTurn();
 
-//  action 3:
+//  action 3: the game calculates scores
+game.calculateScore();
 
 //  7)
-//  Noun:
-//  action 1:
+//  Noun: vehicle
+//  action 1: the vehicle moves
+vehicle.move();
 
-//  action 2:
+//  action 2: the vehicle accelerates
+vehicle.accelerate();
 
-//  action 3:
+//  action 3: the vehicle brakes
+vehicle.brake();
 
 //  8)
-//  Noun:
-//  action 1:
+//  Noun: compiler
+//  action 1: the compiler checks the syntax
+compiler.checkSyntax();
 
-//  action 2:
+//  action 2: the compiler checks the semantics
+compiler.checkSemantics();
 
-//  action 3:
+//  action 3: the compiler builds
+compiler.build();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

Reply:
Made changes as requested